sample/F8/tracks

```diff
@@ -0,0 +1,209 @@
+[{
+		"id": 1,
+		"name": "ADS & MONETIZATION",
```

Comment:
This is malformed JSON. Please pass the JSONs through a formatter and validator. The issue here is that `"` and `"` are not same.

Please refer this - https://jsonformatter.curiousconcept.com/

Paste the tracks json here